de_workloads/ingest/Ingest_AzureSql_Example/data_factory/adf_pipelines.tf

```diff
@@ -0,0 +1,34 @@
+resource "random_uuid" "deployment" {
+}
+
+resource "azurerm_resource_group_template_deployment" "pipeline_Ingest_AzureSql_Example" {
+  #   count           = var.include_data_quality == false ? 1 : 0
+  name                = "Ingest_AzureSql_Example-${random_uuid.deployment.result}"
+  resource_group_name = var.data_factory_resource_group_name
+  deployment_mode     = var.arm_deployment_mode
+  parameters_content = jsonencode({
+    "factoryName" = {
+      value = data.azurerm_data_factory.factory.name
+    }
+  })
+  template_content = file("${path.module}/pipelines/Ingest_AzureSql_Example.json")
+  depends_on = [
+    azurerm_data_factory_custom_dataset.ds_ex_AzureSql_ExampleSource
+  ]
+}
+
+resource "azurerm_resource_group_template_deployment" "pipeline_Ingest_AzureSql_Example_DQ" {
+  #   count           = var.include_data_quality == true ? 1 : 0
```

Comment:
is this comment intentional, I though we wanted to have DQ optional

Reply:
We we having it as optional, then we have decided to comment this out for now so both versions of the example pipeline get deployed. We'll address this in the templating feature.